yt_dlp/extractor/youtube.py

```diff
@@ -5781,6 +5793,9 @@ class YoutubeSearchURLIE(YoutubeTabBaseInfoExtractor):
             'entries': [{
                 'url': r're:https://(www\.)?youtube\.com/hashtag/cats',
                 'title': '#cats',
+                'ie_key': 'YoutubeTab',
+                '_type': 'url',
+                '__x_forwarded_for_ip': None,
```

Comment:
This isnt needed afaik. If test fails without this, it's a bug in the test runner